DEVELOPMENT.md

```diff
@@ -99,6 +100,43 @@ In general most code is expected to work on Android (but might not yet) with the
 * Functionality aimed at the developer environment, such as console and development provider
 * Any JMX related code.
 
+### The ASF / Github Integration
+
+The Quarks code is in ASF resident git repositories.  
+The repositories are mirrored on github:
+
+    https://github.com/apache/incubator-quarks
+
+Use of the normal github workflow brings benefits to the team including 
+lightweight code reviewing, automatic regression tests, etc 
+for both committers and non-committers.  
+
+For a description of the github workflow see:
+
+    https://guides.github.com/activities/hello-world/
+
+In summary:
+    * fork the incubator-quarks github repository
+    * clone your fork, use lightweight per-task branches, and commit / push changes to your fork
+      * name your branch quarks-xxxx - typically xxxx is a Jira ticket number
```

Comment:
github recommends descriptive branch names [1]. While 'quarks-xxxx' is descriptive, but at one level removed, you have to got to jira to figure out what the branch is about. Maybe:

* name your branch, description branch names are good, you can also include a reference to the Jira issue (QUARKS-100) you are working on, e.g. mqtt-ssl-quarks-100

[1] https://guides.github.com/introduction/flow/